lib/analytics/load_service.rb

```diff
@@ -7,7 +7,7 @@ class LoadService
     include Google::Apis::AnalyticsV3
     include Google::Auth
 
-    SCOPES = ["https://www.googleapis.com/auth/analytics.edit"].freeze
+    SCOPES = [AUTH_ANALYTICS_EDIT].freeze
```

Comment:
Where is this constant defined?

Reply:
It's in the `Google::Apis::AnalyticsV3` module:

https://github.com/googleapis/google-api-ruby-client/blob/master/generated/google/apis/analytics_v3.rb#L34